packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -438,7 +438,20 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
         child: AutocompleteHighlightedOption(
           highlightIndexNotifier: _highlightedOptionIndex,
           child: Builder(
-            builder: (BuildContext context) => widget.optionsViewBuilder(context, _select, _options),
+            builder: (BuildContext context) {
+              final Widget options = widget.optionsViewBuilder(context, _select, _options);
+              final RenderBox? fieldRenderBox = _fieldKey.currentContext?.findRenderObject() as RenderBox?;
```

Comment:
Is it guaranteed at this point that the render object is already laid out?

Reply:
I was hoping you would know :) This method is the overlayChildBuilder of an OverlayPortal, and _fieldKey is in the `child` of the same OverlayPortal.

Reply:
If the sizing rule (`BoxConstraints` => `Size`) is known (in other words the child is sized by the parent) I think you could use a layout builder as the ancestor of the text field to get the size of the text field. Otherwise I think it's a bit trickier.

Reply:
Done and it seems to work, thanks.